Tao_Current.txt

```diff
@@ -1,73 +1,84 @@
+Since its early years, attendance at Internet Engineering Task Force (IETF) face-to-face meetings has grown phenomenally and is now stabilizing. Many of the attendees are new to the IETF at each meeting, and many of those go on to become regular attendees. When the meetings were smaller, it was relatively easy for a newcomer to adjust. Today, however, a newcomer meets many more new people, some previously known only as the authors of documents or thought-provoking email messages.
 
-This document describes many aspects of the IETF, with the goal of explaining to newcomers how the IETF works. This will give them a warm, fuzzy feeling and enable them to make the meeting and the Working Group discussions more productive for everyone. This document started out fairly short, but expanded over time in response to suggestions from IETF novices about what more they would have wanted to know before attending their first face-to-face meeting or becoming active in their first Working Group.
+This document describes many aspects of the IETF, with the goal of explaining to newcomers how the IETF works. This will enable them to make the meeting and the Working Group discussions more productive for everyone. This document started out fairly short, but expanded over time in response to suggestions from IETF novices about what more they would have wanted to know before attending their first face-to-face meeting or becoming active in their first Working Group.
 
 Of course, it's true that many IETF participants don't go to the face-to-face meetings at all. Instead, they're active on the mailing list of various IETF Working Groups. Since the inner workings of Working Groups can be hard for newcomers to understand, this document provides the mundane bits of information that newcomers will need in order to become active participants.
 
 The IETF is always in a state of change. Although the principles in this document are expected to remain largely the same over time, practical details may well have changed by the time you read it; for example, a web-based tool may have replaced an email address for requesting some sort of action.
 
-Many types of IETF documentation are mentioned in the Tao, from BCPs to RFCs and STDs. BCPs make recommendations for Best Current Practices in the Internet; RFCs are the IETF's main technical documentation series, politely known as "Requests for Comments"; and STDs are RFCs identified as "standards". Actually, all three types of documents are RFCs; see Section 6 for more information.
+Many types of IETF documentation are mentioned in the Tao, from BCPs to RFCs and STDs. The IETF publishes its main technical documentation as RFC, politely known as "Requests for Comments"; and STDs are RFCs identified as "standards". BCPs make recommendations for Best Current Practices in the Internet and are also RFCs; All three types of documents are in the RFC document stream; see Section 6 for more information.
```

Comment:
You probably don't want to use "stream" here because "stream" has a technical meaning (IETF Stream,e tc.)

This whole "main technical documentation" thing is hard to process in both versions, and I'm not sure what it's trying to say. Does the IETF have some *other* technical documentation venue? I guess you could say "I-Ds". What do we think this is supposed to mean?

Reply:
Many types of IETF documentation are mentioned in the Tao, from BCPs to RFCs and STDs. The IETF publishes its technical documentation as RFCs, politely known as "Requests for Comments"; and STDs are RFCs identified as "standards". BCPs make recommendations for Best Current Practices in the Internet and are also RFCs; All three types of documents are in the RFC document series; see Section 6 for more information.